fire/SFMainMod.F90

```diff
@@ -940,8 +940,8 @@ subroutine  crown_damage ( currentSite )
                    if (currentCohort%crown_FI >= crown_fire_threshold) then ! 200 kW/m = threshold for crown fire potential
                       
                       ! Initiation of passive crown fire, EQ 9 Bessie and Johnson 1995                     
-                      currentCohort%passive_crown_fire = currentPatch%FI/currentCohort%crown_FI
-                      if (currentCohort%passive_crown_fire > 1.0_r8) then
+                      currentCohort%ignite_crown = currentPatch%FI/currentCohort%crown_FI
```

Comment:
@pollybuotte does this make more sense?

Reply:
@jkshuman if the logic seems confusing, I would be fine with removing all the `else`s because you have already initialized `crown_fire = 0` and `fraction_crown_burned = 0` so they will end up with zero if none of the `if`s are true.

Reply:
@jkshuman yes.

Reply:
@slevisconsulting I removed the extra detail, but retained the endif in latest commit